### PR TITLE
Adds a notification when initially entering a SQL shell within SQL session types

### PR DESCRIPTION
**Pull request by cgranleese-r7**
This PR adds a notification when a user enters a SQL shell within SQL session types via the `query_interactive` command.

The aim was to offer some basic information to the users upon entering the shell.

image

## Verification

- [ ] Start `msfconsole`
- [ ] Ensure you have the features set for the new session types - `features set <session type> true`
- [ ] Get a session for Meterpreter, MySQL, MSSQL, PostgreSQL and SMB - `mysql_login`, `mssql_login`, `postgres_login` and `smb_login`
- [ ] **Verify** the notification appears when entering each of the new session types
- [ ] **Verify** code changes are sane

### CODE REVIEW

File: lib/rex/post/sql/ui/console.rb [hunk 1]
```diff
@@ -77,12 +77,30 @@ def help_to_s(opts = {})
             super + format_session_compatible_modules
           end
 
+          #
+          # Notification to display when initially interacting with the client via the query_interactive command
+          #
+          # @return [String]
+          def interact_with_client_notification
+            print_good("Starting interactive SQL shell for #{session.type} @ #{client.peerinfo} (#{current_database})")
```
Comment (adfoster-r7) [inline lib/rex/post/sql/ui/console.rb:85]:
```suggestion
            print_status("Starting interactive SQL shell for #{session.type} @ #{client.peerinfo} (#{current_database})")
```

File: lib/rex/post/sql/ui/console.rb [hunk 2]
```diff
@@ -77,12 +77,30 @@ def help_to_s(opts = {})
...
+            print_good('SQL commands ending with ; will be executed on the remote server. Use the %grnexit%clr command to exit.')
```
Comment (adfoster-r7) [inline lib/rex/post/sql/ui/console.rb:86]:
```suggestion
            print_status('SQL commands ending with ; will be executed on the remote server. Use the %grnexit%clr command to exit.')
```

Comment (adfoster-r7) [reply lib/rex/post/sql/ui/console.rb:86]: `print_line`

File: lib/rex/post/sql/ui/console.rb [hunk 3]
```diff
@@ -77,12 +77,31 @@ def help_to_s(opts = {})
...
+            print_status("Starting interactive SQL shell for #{session.type} @ #{client.peerinfo} (#{current_database})")
+            print_status('SQL commands ending with ; will be executed on the remote server. Use the %grnexit%clr command to exit.')
+            print_line
+          end
+
+          #
+          # Create prompt via client and session data
+          #
+          # @return [String]
+          def prompt
```
[the remark this answers is not in the document]
Comment (cgranleese-r7) [reply lib/rex/post/sql/ui/console.rb:94]: Whoops I had meant to add it above as well. I can just remove it and handle both instances with 
```
#{session.type} @ #{client.peerinfo} (#{current_database}
``` 
If that works for you?

[the remark this answers is not in the document]
Comment (adfoster-r7) [reply lib/rex/post/sql/ui/console.rb:94]: Sounds good; just remember to rename the method to something that doesn't currently exist already :+1:

**Comment by adfoster-r7** [comment]
## Release Notes

Adds additional help information when interacting with a SQL session

**Pull request merged**